lib/zendesk_api/resources.rb

```diff
@@ -500,6 +500,14 @@ class SuspendedTicket < ReadResource
     put :recover
   end
 
+  class TargetFailure < ReadResource
+    def target
+      @client.targets.to_a.find do |target|
+        target.title == self.target_name
```

Comment:
Given the match is done on title, I would suggest naming the method `target_by_title`. (?? not too sure).